### PR TITLE
Many fixes for RTL

### PR DESCRIPTION
**Pull request by fduraibi**
A lot of fixes for RTL

### CODE REVIEW

File: packages/rocketchat-theme/assets/stylesheets/rtl.less [hunk 1]
```diff
@@ -191,22 +212,181 @@
...
+				float: right;
+			}
+		}
+		p {
+			float: right;
+			padding-right: 10px;
+			padding-left: unset;
+		}
 	}
-	q:before {
```
Comment (engelgabriel) [inline packages/rocketchat-theme/assets/stylesheets/rtl.less]: Why did you remove this? I believe it is important for quotes

Comment (fduraibi) [inline packages/rocketchat-theme/assets/stylesheets/rtl.less]: @engelgabriel It was not correct and didn't apply to the blockquote, but I forgot to fix it with this patch.

I have it fixed with other issues I found later and will include them in another patch
